includes/admin/class-wcmypa-admin.php

```diff
@@ -219,14 +220,29 @@ public function displayDeliveredPostStatus(array $order_statuses): array
     }
 
     /**
-     * @param $orderId
+     * @param             $orderId
+     * @param string|null $old_status
+     * @param string|null $new_status will be passed when order status change triggers this method
      *
-     * @throws ErrorException
+     * @throws \ErrorException
      * @throws \MyParcelNL\Sdk\src\Exception\ApiException
      * @throws \MyParcelNL\Sdk\src\Exception\MissingFieldException
      */
-    public function automaticExportOrder($orderId): void
+    public function automaticExportOrder($orderId, ?string $old_status = null, ?string $new_status = null): void
     {
+        if (! WCMYPA()->setting_collection->isEnabled(WCMYPA_Settings::SETTING_AUTOMATIC_EXPORT)){
+            return;
+        }
+
+        $for_status = WCMYPA()->setting_collection->getByName(WCMYPA_Settings::SETTING_AUTOMATIC_EXPORT_STATUS);
```

Comment:
Waarom gebruik je hier snake_case en geen camelCase? Zie ook andere plekken.

Reply:
Sorry is nog macht der gewoonte...